Tests/Clappr_Tests/Classes/Base/PlaybackTests.swift

```diff
@@ -87,6 +87,14 @@ class PlaybackTests: QuickSpec {
 
                 expect(callbackWasCalled) == false
             }
+            
+            it("stops playing when destroy is called") {
+                let playbackSpy = PlaybackSpy(options: options)
+                
+                playbackSpy.destroy()
+                
+                expect(playbackSpy.didCallStop).to(beTrue())
+            }
```

Comment:
I think we can change the tests for the concrete entity. The new scenario (check if isStopped is false) was not covered by tests and the internal changes of `AVFoundationPlayback.stop()` will not reflect the reality on `PlaybackSpy`. Maybe we can change the expectation for one of the triggered events inside stop.
What do you think?